mindsdb_native/libs/controllers/predictor.py

```diff
@@ -54,13 +54,13 @@ def _prepare_sample_settings(user_provided_settings,
 
 def _prepare_timeseries_settings(user_provided_settings):
```

Comment:
up to preference but can you put a docstring here to explain what each of these vars are doing, nr_predictions is not obvious to me but the others are self-evident

Reply:
Will do